framework/include/transfers/MultiAppFieldTransfer.h

```diff
@@ -30,37 +34,22 @@ class MultiAppFieldTransfer : public MultiAppTransfer
 
   virtual void initialSetup();
 
+protected:
   /**
-   * Add some extra work if necessary after execute(). For example, adjust the solution
-   * to preserve some physics quality of interest.
+   * Performs the transfer of a variable between two problems if they have the same mesh.
    */
-  virtual void postExecute();
-
-protected:
-  std::vector<VariableName> _from_var_names;
-  std::vector<AuxVariableName> _to_var_names;
-
-  VariableName _from_var_name;
-  AuxVariableName _to_var_name;
```

Comment:
Ah... So it was Aux in this class. It's hard to keep track of the renames and changes simultaneously.

Reply:
Yes, there were a bunch of shifts in this to do the restructure that Fande and I discussed. But the "to" variable in MultiAppCopyTransfer needed to be an AuxVariable to be able to derive from MultiAppFieldTransfer.

Reply:
A `bunch of shifts` is right...